components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -244,9 +247,22 @@
                         $(jQuery('#idTokenPlain').hide());
                         $(jQuery('#logout_mechanism_row').hide());
                         $(jQuery('#logout_url_row').hide());
-                        $(jQuery("#audience_enable").hide());
-                        $(jQuery("#add_audience").hide());
-                        $(jQuery("#audience_table").hide());
+
+                        if (OAuth2ServiceComponentHolder.isLegacyAudienceEnabled()) {
+                            $(jQuery("#audience_enable").hide());
+                            $(jQuery("#add_audience").hide());
+                            $(jQuery("#audience_table").hide());
+
```

Comment:
remove the unwanted new line. check all such places

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b